control_panel_api/tests/test_services.py

```diff
@@ -19,50 +20,47 @@ def test_policy_document_readwrite(self):
         sids = [s['Sid'] for s in document['Statement']]
         self.assertNotIn('UpdateRenameAndDeleteObjects', sids)
 
-    @patch('boto3.client')
-    def test_create_bucket(self, mock_client):
+    @patch('control_panel_api.aws.put_bucket_logging')
+    @patch('control_panel_api.aws.create_bucket')
+    def test_create_bucket(self, mock_create_bucket, mock_put_bucket_logging):
         services.create_bucket('bucketname')
 
-        mock_client.return_value.create_bucket.assert_called_with(
-            Bucket='test-bucketname',
-            ACL='private',
-            CreateBucketConfiguration={
-                'LocationConstraint': 'eu-test-2',
-            },
-        )
-        mock_client.return_value.put_bucket_logging.assert_called_with(
-            Bucket='test-bucketname',
-            BucketLoggingStatus={
-                'LoggingEnabled': {
-                    'TargetBucket': 'moj-test-logs',
-                    'TargetPrefix': 'test-bucketname/'
-                }
-            }
-        )
-
-    @patch('boto3.client')
-    def test_delete_bucket(self, mock_client):
+        mock_create_bucket.assert_called_with('test-bucketname', region='eu-test-2', acl='private')
+        mock_put_bucket_logging.assert_called_with('test-bucketname', target_bucket='moj-test-logs',
+                                                   target_prefix='test-bucketname/')
+
+    @patch('control_panel_api.aws.delete_bucket')
+    def test_delete_bucket(self, mock_delete_bucket):
         services.delete_bucket('bucketname')
 
-        mock_client.return_value.delete_bucket.assert_called_with(
-            Bucket='test-bucketname',
-        )
+        mock_delete_bucket.assert_called_with('test-bucketname')
 
-    @patch('boto3.client')
-    def test_create_bucket_policies(self, mock_client):
+    @patch('control_panel_api.aws.create_policy')
+    def test_create_bucket_policies(self, mock_create_policy):
         services.create_bucket_policies('bucketname')
 
         expected_calls = [
-            call(
-                PolicyName='test-bucketname-readwrite',
-                PolicyDocument=json.dumps(services.get_policy_document('test-bucketname', True))
-            ),
-            call(
-                PolicyName='test-bucketname-readonly',
-                PolicyDocument=json.dumps(services.get_policy_document('test-bucketname', False))
-            )
+            call('test-bucketname-readwrite', services.get_policy_document('test-bucketname', True)),
```

Comment:
I still think using `services.get_policy_document()` (the real thing), in this tests make them very easy to break. 

Ping me on slack if this sound confusing, I may be able to explain better there.